solr/solr-ref-guide/modules/indexing-guide/pages/post-tool.adoc

```diff
@@ -44,51 +44,59 @@ The basic usage of `bin/solr post` is:
 
 [source,plain]
 ----
-$ bin/solr post -h
```

Comment:
Is there a way to maintain this without duplicating?

Reply:
Not really...     I'd like, in another ticket to go through all the features of the PostTool and really think about which we want to keep and which we want to remove...   Once I do that, I'll write up the options in more detail...

I'd also like to rework a bit the Solr Control Script ref guide page to be better organized as well..